ambry-mysql/src/main/resources/AmbryRepairRequests.ddl

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+/* CREATE DATABASE AmbryRepairRequests; */
+
+/**
+ * This table is created to store the Ambry requests which were partially failed and need to get repaired.
+ */
+CREATE TABLE IF NOT EXISTS ambry_repair_requests (
+    blobId varbinary(50) NOT NULL,
+    partitionId INT NOT NULL,
+    sourceHostName varchar(50) NOT NULL,
+    sourceHostPort int NOT NULL,
+    operationType smallint NOT NULL,
+    operationTime datetime(6) NOT NULL,
+    lifeVersion smallint NOT NULL,
+    expirationTime datetime(6) DEFAULT NULL,
+    PRIMARY KEY (blobId, operationType),
+    INDEX operationTimeIndex (operationTime),
```

Comment:
we probably don't need an index on the operationTime.

Reply:
We may query by operationTime asc order, right?

Reply:
we select by partition id first, right, if we create index, shouldn't we create index on (partitionid, operationTime)?

Reply:
You are right. Fixed.